bedrock/firefox/views.py

```diff
@@ -933,3 +938,14 @@ def firefox_welcome_page1(request):
     template_name = 'firefox/welcome/page1.html'
 
     return l10n_utils.render(request, template_name, context)
+
+
+def firefox_mobile(request):
+    locale = l10n_utils.get_locale(request)
+
+    if locale.startswith('en-'):
```

Comment:
I see above we've added lang files to start localizing this new page, but we still have a hard-coded en conditional here, so right now only English locales will see the new design. 

We could ship this as-is, and remove this conditional later in a followup PR (which is totally fine), or we could change this to an `if lang_file_is_active('firefox/mobile-2019', locale)` conditional, so locales that have the new strings translated will automatically see the new page, falling back to the old design if not.

Note: that if we change this now, you'll probaly also need to remove the functional tests for the old Pebbles themed page, since any locale could switch to the new design independently of a code change in bedrock.

Reply:
done and https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/453/pipeline/